cosmoz-treenode-navigator.js

```diff
@@ -141,13 +144,26 @@
 				value: 1
 			}
 		},
-		_computeDataPlane: function (searching, inputValue, renderedLevel, openNodeLevelPathParts, data) {
+
+		_computeDataPlane: function (searching, inputValue, renderedLevel, openNodeLevelPathParts, tree) {
 			if (searching) {
-				return this.searchAllBranches(inputValue, openNodeLevelPathParts, renderedLevel, data);
+				return tree.getNodeByProperty(this.comparisonProperty, inputValue, false, true, renderedLevel);
 			}
 			return renderedLevel;
 		},
 		/**
+		 * A hook to manipulate the default tree class
+		 * of Cosmoz.TreeBehavior
+		 */
+		_getTreeClass: function (treeClass) {
+			treeClass.prototype.afterFound = function (node, propertyName) {
```

Comment:
Any reason why we can't just do the equivalent in the `map()` of `_renderLevel` ?
This code both updates an unknown tree implementation (once `data` is replaced with `tree`) and it writes to the nodes in the tree.